typescript/scripts/generate-openapi.ts

```diff
@@ -23,27 +23,66 @@ async function generate_types(appName: string) {
   const jsonFile = `./openapi.json`;
   console.log(`Parsing ${jsonFile} to generate typescript files...`);
   const schema = await fs.promises.readFile(jsonFile, "utf8");
+  const schemaContent = JSON.parse(schema);
 
-  const ast = await openapiTS(JSON.parse(schema), {
-    exportType: true,
-    // https://openapi-ts.dev/migration-guide#defaultnonnullable-true-by-default
+  const ast = await openapiTS(schemaContent, {
     defaultNonNullable: false,
-    transform(schemaObject, _options): TypeNode | undefined {
-      if (schemaObject.format === "binary") {
-        if (schemaObject.nullable) {
-          return ts.factory.createUnionTypeNode([BLOB, NULL]);
-        } else {
-          return BLOB;
-        }
+    exportType: true,
+    generatePathParams: true,
+    transform(schemaObject, meta) {
+      if (
+        schemaObject.type === "object" &&
+        schemaObject.additionalProperties &&
+        schemaObject.properties?.empty
+      ) {
+        // Records generated from LanguageFieldDTO have additional boolean property empty
```

Comment:
Denne er for å fjerne boolean-feltet fra generert typescript.

<img width="610" height="224" alt="Image" src="https://github.com/user-attachments/assets/dbfacc76-b9ec-4fc5-9d0b-4f634ff5763f" />

Reply:
Jo, har oppdaert no!